client/src/util/stateManager/colorHelpers.ts

```diff
@@ -85,9 +85,10 @@ export function createColorQuery(
   }
 }
 
-function _defaultColors(nObs: number): ColorTable {
+function _defaultColors(nObs: number, isSpatial: boolean): ColorTable {
+  const DEAFULT_COLOR = isSpatial ? 0.3 : 0;
   return {
-    rgb: new Float32Array(3 * nObs).fill(0),
+    rgb: new Float32Array(3 * nObs).fill(DEAFULT_COLOR),
```

Comment:
Updating default color for spatial from black to `#444444` gray, as mix blend mode does not have any effect on black